VSharp.Explorer/Explorer.fs

```diff
@@ -110,21 +115,32 @@ type private SVMExplorer(explorationOptions: ExplorationOptions, statistics: SVM
                 None
             else
                 Some options.randomSeed
+
         match mode with
         | AIMode ->
-            match options.aiAgentTrainingOptions with
+            let useGPU =
+                options.useGPU.IsSome && options.useGPU.Value
```

Comment:
What is the case when ```useGPU``` is none? Can we just use ```false``` as a default value for this field?